samples-v2/fan_in_fan_out/tests/test_E2_CopyFileToBlob.py

```diff
@@ -0,0 +1 @@
+# Stub file - test this function with standard Azure Functions Python testing tools. 
```

Comment:
Why is this a stub file?

Reply:
I also added test files for every Durable function in all 3 apps, just for consistencey